app/src/androidTest/java/com/github/sdp/tarjetakuna/model/UserTest.kt

```diff
@@ -0,0 +1,157 @@
+package com.github.sdp.tarjetakuna.model
+
+import androidx.test.espresso.matcher.ViewMatchers.assertThat
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.github.sdp.tarjetakuna.database.CardPossession
+import com.github.sdp.tarjetakuna.database.DBMagicCard
+import com.github.sdp.tarjetakuna.database.FirebaseDB
+import com.github.sdp.tarjetakuna.mockdata.CommonMagicCard
+import com.github.sdp.tarjetakuna.utils.FBEmulator
+import com.google.gson.Gson
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.withTimeout
+import org.hamcrest.CoreMatchers
+import org.junit.Assert.assertThrows
+import org.junit.ClassRule
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.Timeout
+import org.junit.runner.RunWith
+import java.util.concurrent.TimeUnit
+
+/**
+ * Tests for [User]
+ */
+@RunWith(AndroidJUnit4::class)
+class UserTest {

```

Comment:
Name is not very explicit

Reply:
This is following the naming convention of the tests: <Class name>Test